index.html

```diff
@@ -1,63 +1,415 @@
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">
+							Back End Dev
+						</p>
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">2015</p>
+					</div>
+					<p class="work-pharagraph">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+					<div class="work-tags">
+						<a class="work-tag" href="#">html</a>
+						<a class="work-tag" href="#">css</a>
+						<a class="work-tag" href="#">javascript</a>
+					</div>
+					<button type="button" class="bg-white-button">See Project</button>
+				</div>
+			</div>
+
+			<div class="work-item">
+				<img
+					src="./assets/img/work/SnapshootPortfolio3.png"
+					alt="tonic"
+					width="295"
+					height="220"
+					class="work-item-image"
+				/>
+				<div class="work-content">
+					<h3>Tonic</h3>
+					<div class="work-header-items">
+						<p class="work-header-item-one work-header-item-margin">CANOPY</p>
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">
+							Back End Dev
+						</p>
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">2015</p>
+					</div>
+					<p class="work-pharagraph">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+					<div class="work-tags">
+						<a class="work-tag" href="#">html</a>
+						<a class="work-tag" href="#">css</a>
+						<a class="work-tag" href="#">javascript</a>
+					</div>
+					<button type="button" class="bg-white-button">See Project</button>
+				</div>
+			</div>
+
+			<div class="work-item">
+				<img
+					src="./assets/img/work/SnapshootPortfolio4.png"
+					alt="Multi-Post Stories"
+					width="295"
+					height="220"
+					class="work-item-image"
+				/>
+				<div class="work-content">
+					<h3>Multi-Post Stories</h3>
+					<div class="work-header-items">
+						<p class="work-header-item-one work-header-item-margin">CANOPY</p>
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">
+							Back End Dev
+						</p>
+						<span class="work-header-item-round work-header-item-margin"></span>
+						<p class="work-header-item-two work-header-item-margin">2015</p>
+					</div>
+					<p class="work-pharagraph">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+					<div class="work-tags">
+						<a class="work-tag" href="#">html</a>
+						<a class="work-tag" href="#">css</a>
+						<a class="work-tag" href="#">javascript</a>
+					</div>
+					<button type="button" class="bg-white-button">See Project</button>
+				</div>
+			</div>
```

Comment:
- [ ] All of the work should be in a ul tag and li for each project

Reply:
Thank you very much. I changed it as you said. :100: